python/cudf/cudf/tests/series/test_conversion.py

```diff
@@ -31,5 +31,18 @@ def test_convert_dtypes(data, dtype):
     assert_eq(expect, got)
 
 
+def test_convert_integer_false_convert_floating_true():
+    data = [1.000000000000000000000000001, 1]
+    expected = pd.Series(data).convert_dtypes(
+        convert_integer=False, convert_floating=True
+    )
+    result = (
+        cudf.Series(data)
+        .convert_dtypes(convert_integer=False, convert_floating=True)
+        .to_pandas(nullable=True)
+    )
+    pd.testing.assert_series_equal(result, expected)
```

Comment:
```suggestion
    assert_eq(result, expected)
```

Wouldn't `assert_eq` work if both are already pandas series?

Reply:
Yeah sure thing we can use this. (I was using `pd.testing.assert_series_equal` just to be explicit)